### DISCUSSION TITLE
How to change osu!lazer install path?

### DISCUSSION THREAD

**Discussion by anispwyn** [ppy/osu#23051] [Q&A]
so basically i have a very small C drive (usually just contains program and necessary stuff)

and my D drive which is made for games and stuff

i want to install osu!lazer on D drive

how do i do that?

**Comment by anispwyn** [comment]
nvm i just found out that you can move osu lazer folder path

**Reply by Iblamemedo** [reply]
Hi i know this is old but i just downloaded the lazer client to try it out and i have the same problem can you please tell me how u did it or which files you moved?
